docs/inputs.rst

```diff
@@ -0,0 +1,35 @@
+Simulation Inputs  
+=================  
+
+This page provides an overview of the nomenclature used for inputs in a Neurodamus circuit simulation. It assumes the reader is already familiar with the NEURON software and its terminology.  
+
+``simulation_config.json``  
+--------------------------  
+
+This is the main SONATA file that controls the simulation settings. Key fields include:  
+
+- What **reports** the simulator should generate.  
+- The duration of the simulation.  
+
+A **report** is one of the possible outputs of the simulation. It typically consists of a time trace of a specific value (e.g., voltage) recorded per neuron. Reports can also be generated per neuronal section.  
+
+``circuit_config.json``  
+-----------------------  
+
+Defines how neurons are structured and connected. It can be seen as a graph of graphs, representing interactions between different neuron populations. This file references:  
+
+- ``nodes.h5``: Contains neuron properties. Note that these *nodes* are unrelated to :class:`neurodamus.node.Node`, which is a low-level simulation orchestrator.  
+- ``edges.h5``: Defines connections between neurons.  
+
+A **population** is a group of neurons that interact within themselves and with other populations. NEURON does not have a concept of populations—this is specific to Neurodamus.  
```

Comment:
It might be worthwhile to link "our" SONATA documentation:
https://sonata-extension.readthedocs.io/en/latest/sonata_tech.html which should hopefully cover most of these topics.

Reply:
added